package.json

```diff
@@ -12,7 +12,7 @@
   "gypfile": true,
   "scripts": {
     "upload": "cross-env prebuild -t $npm_config_node_version -u $NODE_PRE_GYP_GITHUB_TOKEN --include-regex 'bindings\\.node$'",
-    "build": "node-gyp build",
+    "build": "node-gyp configure && node-gyp build",
```

Comment:
You can use `node-gyp rebuild` here, which will do `clean`, `configure` and `build`

Reply:
:+1: I amended my commit with this